### PR TITLE
Add import glue to scripts that import ga4gh

### PR DESCRIPTION
**Pull request by dcolligan**
Issue #1083

### CODE REVIEW

**Comment by macieksmuga** [comment]
Looks like a good fix for now, especially as I'd like to re-think our directory structure anyway (move the API-able parts of scripts off into a module or modules within GA4GH, then have the code in `scripts` just be the front-end calls to these APIs.

File: scripts/prepare_compliance_data.py [hunk 1]
```diff
@@ -214,11 +214,13 @@ def main():
         default=None)
     parser.add_argument('--verbose', '-v', action='count', default=0)
     args = parser.parse_args()
+    cdm = None
```
Comment (jeromekelleher) [inline scripts/prepare_compliance_data.py]: The right way to do this is 
```python
cdm = ComplianceDataMunger(args.inputDirectory, args.outputDirectory)
try:
     cdm.run()
finally:
     cdm.cleanup()
 ```

**Comment by jeromekelleher** [comment]
Very nice, +1, modulo comment above.

**Comment by dcolligan** [comment]
Addressed the comments... merging...

**Pull request merged**